README.md

```diff
@@ -234,6 +234,15 @@ You might, for example, include a constraint in your manifest that specifies `ve
 * `0.2.3` becomes the range `>=0.2.3, <0.3.0`
 * `0.0.3` becomes the range `>=0.0.3, <0.1.0`
 
+## Timeouts
+
+`dep ensure` can sometime fail on very slow system due to underlying operations taking too long to perform. This is especially true on Windows due to filesystem issues.
+
+If the built-in timeouts are not suitable to your environment, you can override those by setting specific environment variables:
+
+* `DEP_EXPENSIVE_CMD_TIMEOUT`: Set the maximum number of seconds expensive operations can take. The default is 2 minutes (120 seconds).
```

Comment:
Can we include more information in the error message when the command times out, telling them which flag they can set to override the timeout? That way it's not hidden in verbose and shows up only when it's needed (when the timeout is hit).